content/posts/fleek-network-node-healthchecks.md

```diff
@@ -145,47 +153,32 @@ For the ones who followed the [getting started guide](#fleek-network-getting-sta
 We execute a `cURL` request with the `--head` or `-I` flag to show the document info only, in our case the headers of our HTTP response.
 
 ```sh
-curl --head 127.0.0.1:4069
+curl --head 127.0.0.1:80
 ```
 
 Or, the shorter version:
 
 ```sh
-curl -I 127.0.0.1:4069
+curl -I 127.0.0.1:80
 ```
 
 The response is:
 
 ```sh
-HTTP/1.1 405 Method Not Allowed
-content-length: 0
-date: Wed, 04 Jan 2023 19:00:52 GMT
+HTTP/1.1 404 Not Found
+Server: nginx/1.23.3
+Date: Fri, 06 Jan 2023 18:36:09 GMT
+Content-Length: 0
+Connection: keep-alive
```

Comment:
```suggestion
HTTP/1.1 200
Server: nginx/1.23.3
Date: Fri, 06 Jan 2023 18:36:09 GMT
Content-Length: 4
Connection: keep-alive
```

or if we want to keep this checkup to headers only, this is what /ping would result wiht